src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -653,6 +661,19 @@ export class ExposureNotificationService {
   }
 
   public shouldPerformExposureCheck = async () => {
+    log.debug({
+      category: 'exposure-check',
+      message: 'shouldPerformExposureCheck',
+      payload: {
+        appState: this.appState,
+      },
+    });
+
+    // pass thru unless the app is open
+    if (this.appState !== 'active') {
```

Comment:
Could we pull this to a helper with a name like `isAppInBackground()` to make it clearer what we're checking?

Reply:
Good idea - we can make a separate PR later to replace other instances as we see fit. 

```javascript
const onAppStateChange = async (newState: AppStateStatus) => {
      if (newState === 'active') {
        await fetchData();
      }
    };
```